src/main/java/edu/ucdavis/dss/ipa/services/jpa/JpaUserRoleService.java

```diff
@@ -226,27 +208,24 @@ public boolean deleteByLoginIdAndWorkgroupId(String loginId, long workgroupId) {
 
 	@Override
 	public List<Long> getInstructorsByWorkgroupIdAndRoleToken(long workgroupId, String roleToken) {
-		String[] INSTRUCTOR_ROLES = {roleToken};
-
 		List<Long> workgroupInstructorIds = new ArrayList<Long>();
 		List<Instructor> workgroupInstructors = new ArrayList<Instructor>();
 
-		for (String instructorRole: INSTRUCTOR_ROLES) {
-			List<UserRole> instructorRoles = this.findByWorkgroupIdAndRoleToken(workgroupId, instructorRole);
-			for (UserRole userRole: instructorRoles) {
-				Instructor instructor = instructorService.getOneByLoginId(userRole.getUser().getLoginId());
-				if (instructor != null) {
-					// Add to list of instructors if not already there. This should never happen since
-					// an instructor should be either Senate OR Federation, but not both.
-					// Prevents getting the AJS dupes error
-					if (!workgroupInstructorIds.contains(instructor.getId())) {
-						workgroupInstructorIds.add(instructor.getId());
-						workgroupInstructors.add(instructor);
-					}
-				} else {
-					Exception e = new Exception("Could not find instructor entity for loginId: " + userRole.getUser().getLoginId());
-					emailService.reportException(e, this.getClass().getName());
+		List<UserRole> instructorUserRoles = this.findByWorkgroupIdAndRoleToken(workgroupId, roleToken);
+
+		for (UserRole userRole: instructorUserRoles) {
+			Instructor instructor = instructorService.getOneByLoginId(userRole.getUser().getLoginId());
+			if (instructor != null) {
+				// Add to list of instructors if not already there. This should never happen since
+				// an instructor should be either Senate OR Federation, but not both.
+				// Prevents getting the AJS dupes error
+				if (!workgroupInstructorIds.contains(instructor.getId())) {
+					workgroupInstructorIds.add(instructor.getId());
+					workgroupInstructors.add(instructor);
 				}
+			} else {
+				Exception e = new Exception("Could not find instructor entity for loginId: " + userRole.getUser().getLoginId());
```

Comment:
I believe you can just throw the exception. They should be e-mailed automatically.

Reply:
Please address this.